src/theme/css/general.css

```diff
@@ -267,7 +269,7 @@ body {
 	width: 100%;
 }
 #page-wrapper {
-	width: 100%;
+	width: calc(100% - 20px);
```

Comment:
This has accidentally fixed the width on mobile for pages with code snippets, in addition to tables.

Before:
<img width="200" alt="screenshot before" src="https://user-images.githubusercontent.com/4586410/108609737-47cde600-73d0-11eb-8ef6-a2abd90aa468.jpeg">

After:
<img width="200" alt="screenshot after" src="https://user-images.githubusercontent.com/4586410/108609752-646a1e00-73d0-11eb-9b6d-f331455710a2.jpeg">